IceCubesApp/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -281,6 +281,7 @@
 "status.editor.drafts.navigation-title" = "Drafts";
 "status.editor.error.upload" = "Error uploading";
 "status.editor.language-select.navigation-title" = "Select Language";
+"status.editor.language-select.recently-used" = "Recently Used";
```

Comment:
Please copy this line to the other languages that the app provides. Then, the language updaters will see the English and create a PR to _fix_ it.

Thanks!

Reply:
Yep, thanks! This need to be duplicated in every locale files or else the raw value  (key) will show.

Reply:
I'm on it